src/partitioner/dask_runner.py

```diff
@@ -1,24 +1,43 @@
-"""Partitioner runner that uses dask for parallelization"""
+"""Partitioner runner that doesn't use any parallelization mechanism"""
```

Comment:
kind of confused about what this means, seems like we still are using dask and maybe I'm just ignorant but wouldn't map reduce be considered a type of parallelization?

Reply:
That looks like an unfortunate copy/paste problem. This runner definitely uses parallelism.